src/main/scala/elo/package.scala

```diff
@@ -31,11 +31,7 @@ package object elo {
   def formatScore(userScore: Double): String = f"$userScore%1.2f"
 
   // =========================================
-  // Type ALL the things!
+  // Implicit conversion functions
 
-  implicit class SlackUserId(userId: String) {
-    // test of whether an ID mentioned is a valid player
-    // slack user IDs are prefixed with "UD"
-    def isValid: Boolean = userId.contains("UD")
-  }
+  implicit def toSlackUserId(s: String): SlackUserId = SlackUserId(s)
```

Comment:
Don't we need to update all the places where we do string interpolation with SlackUserIds?  Challenge message, leaderboard, etc

Reply:
We won't need to update everywhere, but you did remind me to add in a toString override on SlackUserId. That should do the trick to render the user id properly in strings.

Reply:
Yeah that's what I meant I guess.  Thanks!